CMakeLists.txt

```diff
@@ -299,7 +299,7 @@ if(USE_SHARED_CMARK)
   find_path(CMARK_INCLUDE_DIRS NAMES cmark.h)
 else()
   add_definitions(-DCMARK_STATIC_DEFINE)
-  set(CMARK_LIBRARIES libcmark_static)
+  set(CMARK_LIBRARIES ${PROJECT_BINARY_DIR}/lib/libcmark.a)
```

Comment:
Does it work on Windows? (where the extension is `.lib` instead of `.a`). Also why we use libcmark and not libcmark_static?